app/templating/summary_context.py

```diff
@@ -21,3 +23,17 @@ def build_summary_rendering_context(schema, schema_json, answer_store, metadata)
             groups.extend([Group(group, path, answer_store, metadata, url_for)])
 
     return groups
+
+
+def build_section_summary_rendering_context(schema, schema_json, answer_store, metadata, current_location):
```

Comment:
this seems very common with `build_summary_rendering_context` consider refactoring